report/2.Analysis/EmbeddedPlatforms.tex

```diff
@@ -1,41 +1,42 @@
 % !TeX root = ../main.tex
 % This section introduces the various embedded platforms that are available on the 5th semester.
+
+%TODO An introduction to the analysis would be kinda cool before the first section.
```

Comment:
An introduction to the analysis would be kinda cool before the first section.
But the introduction is kinda after the section header?
How do you guys usually do this? We used to have text before the first header.

Reply:
smid noget ind. "Vi vil i dette kapitel undersøge de forskellige relevante teknologier og emner som kan gavne det nævnt i vores introduktion. dette vil bruges til at komme frem til et konkret problem stilling og nogle krav. blabla. her nednævrer vi problem.. bla"